<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVMParametersDialog</class>
 <widget class="QDialog" name="SVMParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="reqC">
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="reqGamma">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>SVM Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="GammaDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kernel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="CDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="reqP">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="PDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="KernelComboBox"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SVMParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SVMParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SVMParametersDialog</receiver>
   <slot>updateReqLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KernelComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SVMParametersDialog</receiver>
   <slot>updateReqLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateReqLabels(int)</slot>
 </slots>
</ui>
